index.html

```diff
@@ -0,0 +1,1681 @@
+                                    </a></div>
+                                <div class="infoz"> <a href="/white-house-supports-covington-maga-hat-students-report-1301124">White
+                                        House Supports Covington MAGA Hat Students</a></div>
+                            </li>
+                            <li class="flex-xs">
+                                <div class="image"><a href="/worlds-best-countries-us-not-top-five-1300813">
+                                        <picture class="mapping-small lazysize">
+                                            <!--[if IE 9]><video style="display: none;"><![endif]-->
+                                            <source data-srcset="https://s.newsweek.com/sites/www.newsweek.com/files/styles/sm/public/2019/01/22/gettyimages-1085348980_0.jpg 1x"
+                                                media="(min-width: 768px)" srcset="https://s.newsweek.com/sites/www.newsweek.com/files/styles/sm/public/2019/01/22/gettyimages-1085348980_0.jpg 1x">
+                                            <source data-srcset="https://s.newsweek.com/sites/www.newsweek.com/files/styles/tablet/public/2019/01/22/gettyimages-1085348980_0.jpg 1x"
+                                                media="(min-width: 481px)" srcset="https://s.newsweek.com/sites/www.newsweek.com/files/styles/tablet/public/2019/01/22/gettyimages-1085348980_0.jpg 1x">
+                                            <source data-srcset="https://s.newsweek.com/sites/www.newsweek.com/files/styles/md_thumb/public/2019/01/22/gettyimages-1085348980_0.jpg 1x"
+                                                media="(min-width: 0px)" srcset="https://s.newsweek.com/sites/www.newsweek.com/files/styles/md_thumb/public/2019/01/22/gettyimages-1085348980_0.jpg 1x">
+                                            <!--[if IE 9]></video><![endif]--><img  class="mapping-small lazysize latest-pic lazyloaded"
+                                                src="https://s.newsweek.com/sites/www.newsweek.com/files/styles/sm/public/2019/01/22/gettyimages-1085348980_0.jpg"
+                                                alt="GettyImages-1085348980" title=""></picture>
+                                    </a></div>
+                                <div class="infoz"> <a href="/worlds-best-countries-us-not-top-five-1300813">U.S.
+                                        Fails to Make the Top 5 World's 'Best' Countries</a></div>
+                            </li>
+
+                            <li class="flex-xs">
+                                <div class="image"><a href="/tennessee-state-lawmaker-proposes-dress-code-bill-parents-visiting-schools-1301101">
+                                        <picture class="mapping-small lazysize">
+                                            <!--[if IE 9]><video style="display: none;"><![endif]-->
+                                            <source data-srcset="ht
```

Comment:
You can get this section out of the `div`. Also, work on mirroring the link structure of the original page and adding the social media icons.